transitions/registration.js

```diff
@@ -313,17 +313,15 @@ module.exports = {
             patientId = doc.patient_id,
             patientNameField = _.first(options.params) || 'patient_name';
 
-        utils.getRegistrations({
-            db: db,
-            id: patientId
-        }, function(err, registrations) {
-            if (err) {
+        db.medic.get('patient-' + patientId, function(err, patientContact) {
```

Comment:
I would pull this out into a utility function which handles the 404 and the ID concatenation.